### PR TITLE
React1 week2

### PR DESCRIPTION
**Pull request by KristinaGru**

### CODE REVIEW

File: react/react1/week2/todo-app/src/App.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import TodoList from './components/TodoList';
+import { todos } from './todos';
+import { useState } from 'react';
+import { v4 as id } from 'uuid';
+
+function App() {
+  const [todoList, setTodoList] = useState(todos);
+
+  function addTodo(description) {
+    if (description.trim()) {
```
Comment (YurijPaharev) [inline react/react1/week2/todo-app/src/App.js:14]: Perfect precaution!

File: react/react1/week2/todo-app/src/App.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      setTodoList([...todoList, { id: id(), description }]);
```
Comment (YurijPaharev) [inline react/react1/week2/todo-app/src/App.js:15]: Wonderful solution, but please notice that you are trimming your description only on the condition check and not in the setting of new TODO

File: react/react1/week2/todo-app/src/components/Timer.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+const Timer = () => {
+  const [seconds, setSeconds] = useState(0);
+  useEffect(() => {
+    setTimeout(() => {
+      setSeconds((prev) => prev + 1);
+    }, 1000);
+  }, [seconds]);
+
+  return <p>You have spent {Math.floor(seconds)} seconds on this website</p>;
```
Comment (YurijPaharev) [inline react/react1/week2/todo-app/src/components/Timer.js:11]: Why do you need to round your seconds?

**Review by YurijPaharev — APPROVED** [review]
Very nice!

File: react/react1/week2/todo-app/src/components/Timer.js (hunk 3 above)
Comment (KristinaGru) [reply react/react1/week2/todo-app/src/components/Timer.js:11]: Leftover from messing around with the counter, I'll remove it 🙈
Thanks a lot for reviewing!

**Pull request merged**